<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Replace</class>
 <widget class="QDialog" name="Replace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>바꾸기</string>
  </property>
  <widget class="QLabel" name="label_Find">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>56</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>찾을 내용</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Find">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>다음 찾기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Replace">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>바꾸기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Find">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Replace">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>56</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>바꿀 내용</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Replace">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Arround">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>주위에 배치</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Upper_Lower">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>대/소문자 구분</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancle">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>취 소</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
